problem1/problem1.rb

```diff
@@ -0,0 +1,16 @@
+class Problem1
+	attr_accessor :result,:array
```

Comment:
Do `array` and `result` need to writable from outside the class? If not consider updating your class signature accordingly.